lib/view/ui/userdatawizard/personalinfo_page.dart

```diff
@@ -65,10 +81,65 @@ class PersonalInfoPage extends StatelessWidget {
               CustomSizedBox(),
               ElevatedButton(
                 key: Key('nextButton'),
-                onPressed: () {
+                style: ElevatedButton.styleFrom(
+                  backgroundColor: Theme.of(context).colorScheme.secondary,
+                ),
+                onPressed: () async {
                   if (controller.formKey.currentState!.validate()) {
-                    // Validation passed, navigate to the next step
-                    wizardController.goToNextStep();
+                    final userData = collectUserData();
+                    String? userFirebaseUid; // Initialize to null
+
+                    final user = FirebaseAuth.instance.currentUser;
+                    if (user != null) {
+                      userFirebaseUid = user.uid;
+
+                      // Check if an image has been selected
+                      if (controller.selectedImage.value != null) {
+                        final photoURL = await _firebaseService.updateUserPhotoURL(controller.selectedImage.value!, userFirebaseUid);
+                        if (photoURL != null) {
+                          userData['photoURL'] = photoURL; // Update userData with photoURL
+                        } else {
+                          // Handle the case where image upload fails
```

Comment:
make Toast Message gobal

Reply:
should i make string constants to be used in the toast or i should let it remain like this and enter data in string??